packages/smart-accounts/src/alchemyEstimateFeesPerGas.ts

```diff
@@ -0,0 +1,100 @@
+import { getBlock } from "viem/actions";
+import {
+  fromHex,
+  type Client,
+  rpcSchema,
+  isHex,
+  type Transport,
+  type Chain,
+  type Account,
+} from "viem";
+import type {
+  UserOperationRequest,
+  SmartAccount,
+} from "viem/account-abstraction";
+import { BaseError } from "@alchemy/common";
+import { bigIntMultiply } from "@aa-sdk/core";
```

Comment:
we shouldn't be importing from @aa-sdk/core in v5, so we should move this to a new pkg or add a TODO.